src/App.tsx

```diff
@@ -49,16 +44,16 @@ function App() {
         <>
             <HelmetProvider>
                 <Helmet htmlAttributes={{ theme: currentTheme }}></Helmet>
-                <Header
-                    activePage={activePage}
-                    handleActivePage={handleActivePage}
-                    totalProductsInCart={totalProductsInCart}
-                    theme={currentTheme}
-                    setTheme={handleTheme}
-                />
-                {activePage === 'about' && <AboutMe />}
-                {activePage === 'products' && <ProductWrapper theme={currentTheme} updateTotalCart={updateTotalCart} />}
-                <Footer />
+                <Routes>
+                    <Route
+                        path={'/'}
+                        element={<LayoutComponent totalProductsInCart={totalProductsInCart} theme={currentTheme} setTheme={handleTheme} />}
+                    >
+                        <Route index element={<AboutMe />} />
+                        <Route path={'/products'} element={<ProductWrapper theme={currentTheme} updateTotalCart={updateTotalCart} />} />
```

Comment:
Do you have any plans to implement `Dynamic Route for ProductPage` to accomplish `Advance Task`

Reply:
I was hoping to finish that task asap and take care of Product Page Later